tests/src/test/scala/cats/tests/FoldableSuite.scala

```diff
@@ -192,6 +192,20 @@ abstract class FoldableSuite[F[_]: Foldable](name: String)(implicit ArbFInt: Arb
     }
   }
 
+  test(s"Foldable[$name].maximumBy/minimumBy") {
```

Comment:
sorry I should've mentioned this in the earlier comment. Can we make these laws like the other `_Ref` laws in Foldable Laws and Reducible laws?

Reply:
Hmm I'm not sure what you mean :thinking: couldn't find `_Ref` anywhere in the file.

Reply:
Ah ok I found them. Ok I will add some laws. Should I keep the tests in the suite?

Reply:
I think having them in law is enough. sorry I should've been clear about where the law code is.

Reply:
Hmm, if I want to add tests about minimum/maximum to `FoldableTests` that would mean adding an `Order` constraint on `B` in `def foldable[A: Arbitrary, B: Arbitrary]`

Reply:
Ah, right. Then we can't add to laws. We can only use the current tests